### PR TITLE
Perf(QTM-494): Replacement of the material by the mui

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-494

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Doc (yarn storybook)
- [ ] Build
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile

### CODE REVIEW

**Review by alizeleal — DISMISSED** [review]

**Review by alizeleal — COMMENTED** [review]
- [x] Coverage (coverage status should not regress)
- yarn test:coverage

- [x]  Unit tests (yarn test:components)
- [x] Regression
- first start the storybook for regression tests(and keep it open):  yarn test:regression:storybook;
- then run the regression tests: yarn test:regression
- [x]  Doc (yarn storybook)
- [x]  Build


 Layout review
- [x]  Chrome

**Review by rapahaeru — CHANGES_REQUESTED** [review]
[Comment removed]

File: components/RangeSlider/RangeSlider.jsx [hunk 1]
```diff
@@ -53,33 +53,40 @@ const RangeSlider = props => {
 
   const StyledSlider = withStyles({
     root: {
-      color: primary[600],
+      color: `${primary[700]} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:56]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 2]
```diff
@@ -53,33 +53,40 @@ const RangeSlider = props => {
...
       height: 8,
     },
     thumb: {
-      backgroundColor: disabled ? neutral[500] : primary[700],
+      backgroundColor: `${disabled ? neutral[500] : primary[700]} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:60]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 3]
```diff
@@ -53,33 +53,40 @@ const RangeSlider = props => {
...
+        boxShadow: 'none !important',
+      },
     },
     valueLabel: {
       width: 'auto',
       top: '-50px',
-      left: halfMeasure,
+      left: `${halfMeasure} !important`,
+      padding: '0px !important',
+      backgroundColor: `${disabled ? neutral[500] : primary[700]} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:79]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 4]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
     },
     markLabel: {
-      marginTop: `${xsmall}px`,
+      marginTop: `${xxsmall}px`,
     },
     track: {
-      height: 8,
-      borderRadius: 4,
-      backgroundColor: hexToRgba(primary[700], 0.5),
+      height: `8px !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:126]: do we need all this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 5]
```diff
@@ -53,33 +53,40 @@ const RangeSlider = props => {
...
       },
+      '&::before': {
+        boxShadow: 'none !important',
+      },
     },
     valueLabel: {
       width: 'auto',
       top: '-50px',
-      left: halfMeasure,
+      left: `${halfMeasure} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:77]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 6]
```diff
@@ -53,33 +53,40 @@ const RangeSlider = props => {
...
+      padding: '0px !important',
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:78]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 7]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
     markLabel: {
-      marginTop: `${xsmall}px`,
+      marginTop: `${xxsmall}px`,
     },
     track: {
-      height: 8,
-      borderRadius: 4,
-      backgroundColor: hexToRgba(primary[700], 0.5),
+      height: `8px !important`,
+      borderRadius: `4px !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:127]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 8]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
+      border: 'none !important',
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:128]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 9]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
+      backgroundColor: `${hexToRgba(primary[700], 0.5)} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:129]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 10]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
     },
     rail: {
-      height: 8,
-      borderRadius: 4,
+      height: `8px !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:132]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 11]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
+      borderRadius: `4px !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:133]: do we need this important?

File: components/RangeSlider/RangeSlider.jsx [hunk 12]
```diff
@@ -113,21 +120,21 @@ const RangeSlider = props => {
...
+      color: `${disabled ? neutral[500] : primary[700]} !important`,
```
Comment (rapahaeru) [inline components/RangeSlider/RangeSlider.jsx:134]: do we need this important?

**Review by rapahaeru — CHANGES_REQUESTED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Doc (yarn storybook)
- [x] Build
- [ ] Code review

### Browsers review
- [x] Layout review
  - [x] Chrome
  - [x] Safari
  - [x] Mobile

**Review by rapahaeru — DISMISSED** [review]

**Review by rapahaeru — APPROVED** [review]

**Review by alizeleal — APPROVED** [review]
- [x] Coverage (coverage status should not regress)

- yarn test:coverage
 

- [x] Unit tests (yarn test:components)

 

- [x] Regression

- first start the storybook for regression tests(and keep it open):  yarn test:regression:storybook;
- then run the regression tests: yarn test:regression

- [x]  Doc (yarn storybook)
- [x]  Build

File: components/RangeSlider/RangeSlider.jsx (hunk 1 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 2 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 3 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 4 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 5 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 8 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 9 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 10 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 11 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

File: components/RangeSlider/RangeSlider.jsx (hunk 12 above)
Comment (MarcosViniciusPC) [reply components/RangeSlider/RangeSlider.jsx]: It's not necessary, I removed it

**Review by rapahaeru — DISMISSED** [review]

**Review by alizeleal — APPROVED** [review]
- [x] Coverage (coverage status should not regress)

- yarn test:coverage
 

- [x] Unit tests (yarn test:components)

 

- [x] Regression

- first start the storybook for regression tests(and keep it open):  yarn test:regression:storybook;
- then run the regression tests: yarn test:regression

- [x]  Doc (yarn storybook)
- [x]  Build

**Pull request merged**